### ISSUE TITLE
FBX Files not Importing

### ISSUE THREAD

**Issue by ockerdave** [TheGameCreators/GameGuruRepo#4955]
Pipeline for Makehuman character (1) Makehuman (2) Mixamo (3) Game Guru Max. I had successfully imported three
Makehuman characters into Game Guru Max last week, but one week later I had no success, window stated FBX is not 
supported in Max. Suggest converting the model file to latest export version. Can you help me with any changes you
may have made to GGM, if none could you give me the version number of your FBX import.
I realise that this may not be important to you, because Characters are not from GGM, but as there is really no RPG
Characters, a bit strange having soldiers giving Quests out.

Of course the problem could lie with either Makehuman or Mixamo, but I did get three characters successfully 
animated and working into GGM.

Keep up the great work, GGM is a great development Base for games.
Many Thanks.

**Comment by synchromesh62** [comment]
> Pipeline for Makehuman character (1) Makehuman (2) Mixamo (3) Game Guru Max. I had successfully imported three Makehuman characters into Game Guru Max last week, but one week later I had no success, window stated FBX is not supported in Max. Suggest converting the model file to latest export version. Can you help me with any changes you may have made to GGM, if none could you give me the version number of your FBX import. I realise that this may not be important to you, because Characters are not from GGM, but as there is really no RPG Characters, a bit strange having soldiers giving Quests out.
> 
> Of course the problem could lie with either Makehuman or Mixamo, but I did get three characters successfully animated and working into GGM.
> 
> Keep up the great work, GGM is a great development Base for games. Many Thanks.

I tried this 5 mins ago with MakeHuman as i have previously done a few and mine imported without issue
Have you tried verifying your installation via Steam then rebooting. 
Always worth a try if you havent.

**Comment by ockerdave** [comment]
Hi synchromesh62.  Thank you for your prompt reply, I Verified my Max file's, all reported ok, also rebooted, but still the same
problem. I know you are a busy man but could you tell me the Makehuman version number that you have, also did you make
your test from a new human then Makehuman to Maximo then GGM.
Thanks for your help once again.

**Comment by synchromesh62** [comment]
> Hi synchromesh62. Thank you for your prompt reply, I Verified my Max file's, all reported ok, also rebooted, but still the same problem. I know you are a busy man but could you tell me the Makehuman version number that you have, also did you make your test from a new human then Makehuman to Maximo then GGM. Thanks for your help once again.

I am Using version 1.2.0 
Could you try something .. Create a quick character in Make Human .. Export as .fbx and then try to import the ,fbx bypassing the Mixamo process ... I want to check if you get the error with the Raw .fbx file from Make Human gives you the same message

**Comment by ockerdave** [comment]
I deleted the Makehuman software completely then downloaded it again.  I created a Character in Makehuman, exported it to GGM and it imported ok apart from animation, then processed in Mixamo, it imported but character only had head, hands & feet like the invisible man. I had used a skin from the Makehuman repository, processed the same character again using the skin from the actual Makehuman Geometries, and all worked perfectly. One action I took after downloading the 1.2 version
was the next day downloaded the daily update. Two things to look at daily update error or repository error.
I have had computers since the Texas Instruments 994A in 1981 always had bugs in software forever.
Thanks again I will carry on regardless.
